src/src/components/list-menu/ListMenu.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <ul :class="customClass">
+    <ListMenuItem v-for="item in menuItems"
+    :key="item.id"
+    :id="item.id"
+    :name="item.name"
+    :icon="item.icon"
+    :toggler="showText"></ListMenuItem>
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 18 spaces but found 4 spaces.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
                  :toggler="showText"></ListMenuItem>
```